config/pmd/custom-ruleset.xml

```diff
@@ -48,4 +48,12 @@
     </properties>
   </rule>
   <!--  END CODE DOCUMENTATION -->
+
+  <!-- DESIGN -->
+  <rule ref="rulesets/design.xml/ExcessiveParameterList">
+    <properties>
+      <property name="maxParameters" value="10"/>
```

Comment:
I think would be better to add ignore for this check in the line this is happening now, instead of just allow 10 params, what do you think? Because for constructor we need that, but only there. And we can disable pmd only in the contructor line.

Reply:
Removed this change to the rule and added the ignore statement to the cnstructor